server/routes/serviceProviderReferrals/serviceProviderReferralsController.ts

```diff
@@ -548,17 +549,51 @@ export default class ServiceProviderReferralsController {
     return ControllerUtils.renderWithLayout(res, view, serviceUser)
   }
 
+  async startEditingActionPlanSession(req: Request, res: Response): Promise<void> {
+    const draft = await this.draftsService.createDraft('actionPlanSessionUpdate', null, {
+      userId: res.locals.user.userId,
+    })
+
+    res.redirect(
+      `/service-provider/action-plan/${req.params.id}/sessions/${req.params.sessionNumber}/edit/${draft.id}/details`
+    )
+  }
+
+  async backwardsCompatibilityEditActionPlanSession(req: Request, res: Response): Promise<void> {
```

Comment:
If you haven't already, would you mind adding a Trello card to remind us to remove this in future? I feel like it's something we'll forget once you're off the project 😿

Reply:
Yup, I'd [already done this](https://trello.com/c/MFA9lGgc/48-remove-backwardscompability-routes-from-action-plan-scheduling-journey) but forget to mention it here!